tools/delete_candidate.php

```diff
@@ -189,6 +209,9 @@ function deleteCandidate($CandID, $PSCID, $confirm, $printToSQL, $DB, $output)
         //delete from parameter_candidate
         $DB->delete("parameter_candidate", array("CandID" => $CandID));
 
+        //delete from SNP_candidate_rel
```

Comment:
@PapillonMcGill should we even be deleting entries in `participant_status_history` if there is not a foreign key relation to `candidate`? I feel like the history should be kept to at least show that a candidate used to exist but is now deleted

Reply:
I believe that the script to delete a candidate is for when a candidate ask that we delete all information related to him. There is the possibility to inactivate a candidate and keep all information related to him. So I would delete info there too but maybe you could ask someone with more knowledge of Loris than me, like @driusan or @samirdas

Reply:
@PapillonMcGill ill keep as is for now